index.html

```diff
@@ -36,6 +36,191 @@ <h2 class="lets-connect">LET'S CONNECT</h2>
+                </ul>
+
+                <p class="more-info">A daily selection of privately 
+                    personalized reads; no accounts or sign-ups required.
+                </p>
+
+                <ul class="languages">
+                    <li><button class="lang" id="lan-1">html</button></li>
+                    <li><button class="lang" id="lan-2">css</button></li>
+                    <li><button class="lang" id="lan-3">javaScript</li></button></li>
+                </ul>
+
+                <button class="see" id="see-1">See Project</button>
+            </div>   
+        </div>
+
+        <div class="works works4">
+            <div class="w_ctn">
+                <img src="images/Snapshoot Portfolio (3).svg" alt="" class="image4">
+                <h2 class="header">Multi-Post</h2>
+
+                <ul class="description">
+                    <li class="canopy">CANOPY</li>
+                    <li><img src="images/Counter.svg" alt="" id="counter"></li>
+                    <li class="back">Back End Dev</li>
+                    <li><img src="images/Counter.svg" alt="" id="counter"></li>
+                    <li class="back">2018</li>
+                </ul>
+
+                <p class="more-info">A daily selection of privately 
+                    personalized reads; no accounts or sign-ups required.
+                </p>
+
+                <ul class="languages">
+                    <li><button class="lang" id="lan-1">html</button></li>
+                    <li><button class="lang" id="lan-2">css</button></li>
+                    <li><button class="lang" id="lan-3">javaScript</li></button></li>
+                </ul>
+
+                <button class="see" id="see-1">See Project</button>
+            </div>
+            <section class="illustration">
+            </section>
+        </div>
+
+        
+    </section>
+
+
+    <section class="about-myself" id="about-myself">
+        <div class="myself">
+            <div>
+                <h2 class="about">About Myself</h2>
+                <p class="experience">Hello I'm a software developer! 
+                        I can help you build a product , feature or website Look 
+                        through some of my work and experience! If you like what you 
+                        see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+
+                <h3 class="connect-1">LET'S CONNECT</h3>
+
+                <ul class="icons-1">
+                    <li><a href="#"><img src="images/Vector (4).svg" class="icons-2"></a></li>
+                    <li><a href="#"><img src="images/Group.svg" class="icons-2"></a></li>
+                    <li><a href="#"><img src="images/Vector (5).svg" class="icons-2"></a></li>
+                    <li><a href="#"><img src="images/Vector.svg" class="icons-2"></a></li>
+                    <li><a href="#"><img src="images/Vector (3).svg" class="icons-2"></a></li>
+                </ul>
```

Comment:
[OPTIONAL] Good job by making the changes requested by the previous reviewer 👍🏼  and I would ask you please to reduce the spacing between this list items to make it match the design in figma 

![image (30)](https://github.com/MarkYegon7/Mark-Portfolio/assets/59707859/47155e27-f99a-4209-ac87-9fa1c711d6b9)